caddytls/setup.go

```diff
@@ -164,6 +164,22 @@ func setupTLS(c *caddy.Controller) error {
 					return c.Errf("Unsupported Storage provider '%s'", args[0])
 				}
 				config.StorageProvider = args[0]
+
+			case "http2":
+				args := c.RemainingArgs()
+				if len(args) != 1 {
+					return c.ArgErr()
+				}
+
+				switch args[0] {
+				case "off":
+					config.DisableHTTP2 = true
+				case "on":
+					config.DisableHTTP2 = false
```

Comment:
No, there isn't. That's exactly why I've introduced DisableHTTP2 (and used golang default boolean value 'false')